tools/editorconfig/.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{yml,twig,php}]
+indent_size = 4
+
+[.travis-ci.yml]
+indent_size = 2
+
+[composer.json]
+indent_size = 4
```

Comment:
This is dangerous because if a new filetype is added that have, for example, 8 spaces of indent by convention, this will be overridden by editorconfig.